File: devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/core/examples/commandmode-example/java/src/main/java/org/acme/{main.class-name}.tpl.qute.java
package org.acme;

import javax.enterprise.context.control.ActivateRequestContext;
import javax.inject.Inject;

import io.quarkus.runtime.Quarkus;
import io.quarkus.runtime.QuarkusApplication;
import io.quarkus.runtime.annotations.QuarkusMain;

@QuarkusMain
public class {main.class-name} implements QuarkusApplication {

    @Override
    public int run(String... args) throws Exception {
        final String name = args.length > 0 ? String.join(" ", args) : "{greeting.default-name}";
        System.out.println("{greeting.message} " + name);
        return 0;
    }
}

File: devtools/platform-descriptor-json/src/main/resources/codestarts/quarkus/core/examples/commandmode-example/kotlin/src/main/kotlin/org/acme/{main.class-name}.tpl.qute.kt
package org.acme

import javax.inject.Inject

import io.quarkus.runtime.QuarkusApplication
import io.quarkus.runtime.annotations.QuarkusMain

@QuarkusMain
class {main.class-name}: QuarkusApplication {

    override fun run(vararg args: String?): Int {
        val name =  if (args.isNotEmpty()) args.joinToString(",") else "{greeting.default-name}"
        println(name)
        return 0
    }

}
